functions/default.pp

```diff
@@ -0,0 +1,10 @@
+function postgresql::default(
+  String $parameter_name
+){
```

Comment:
Can we improve the wording of what this function does,  for yarrdoc/puppet strings. Otherwise 👍  
```
 ## Functions

+### postgresql::default
+
+Type: Puppet Language
+
+The postgresql::default function.
+
+#### `postgresql::default(String $parameter_name)`
+
+The postgresql::default function.
+
+Returns: `Any`
+
+##### `parameter_name`
+
+Data type: `String`
+
+
+
```

Reply:
To generate the docs
``` bundle exec puppet strings generate --format markdown --out REFERENCE.md ```

Reply:
@tphoney I can't find an example of how to add docs to a puppet function.  I checked a few supported modules and I don't see any puppet functions... also PDK doesn't have a `pdk new function` which I thought might provide me a good template.  

I took the comment section from the top of a new defined type and put it up there but it doesn't seem to get read when generating the docs.  Can you help me out?

Reply:
https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/lib/puppet/functions/type_of.rb is a good example.  following from https://puppet.com/docs/puppet/6.7/puppet_strings_style.html

Reply:
@npwalker You are just missing a bash on the last previous to the `function` declaration. Strings style guide recommends  max 140 chars per line so something like the following should do the trick:
```
# @summary 
#   This function pull default values from the params or globals classes as appropriate
#
# @example
#   postgresql::default('variable')
#
function postgresql::default(
```